Assets/HoloToolkit/Input/Scripts/Voice/SpeechInputSource.cs

```diff
@@ -39,13 +39,14 @@ public enum RecognizerStartBehavior { AutoStart, ManualStart };
         [Tooltip("Whether the recognizer should be activated on start.")]
         public RecognizerStartBehavior RecognizerStart;
 
-        [Tooltip("The confidence level for the keyword recognizer.")]
-        public ConfidenceLevel RecognitionConfidenceLevel;
-
         [Tooltip("The keywords to be recognized and optional keyboard shortcuts.")]
         public KeywordAndKeyCode[] Keywords;
 
 #if UNITY_WSA || UNITY_STANDALONE_WIN
+        [Tooltip("The confidence level for the keyword recognizer.")]
+        //The serialized data of this field will be lost when switching between platforms and re-serializing this class.
+        public ConfidenceLevel RecognitionConfidenceLevel;
```

Comment:
Why don't we go ahead and set a default level?

Reply:
High is the default,  which was also the hard coded level before it got extracted into the field.

Reply:
Where?

Reply:
Per default value

Reply:
I think we should go ahead and set it here to minimize the effect of reserialization.